project/Build.scala

```diff
@@ -382,9 +382,16 @@ object Build {
       )
       .dependsOn(junitAsyncJVM % "test")
 
+  import scala.scalanative.build._
+
   lazy val sandbox =
     MultiScalaProject("sandbox", file("sandbox"))
       .enablePlugins(MyScalaNativePlugin)
+      .settings(nativeConfig ~= {
+        _.withLTO(LTO.default)
+          .withMode(Mode.default)
+          .withGC(GC.default)
+      })
```

Comment:
@LeeTibbert Contributors can go here and temporarily change settings as they wish. That is it. The change is just to add it here so you don't have to look around or remember this if you need some different settings.

Reply:
Thank you, now I understand.  I appreciate the example.